README.md

```diff
@@ -222,6 +222,28 @@ In order to do that, just call:
 openAds.resetConnectors()
 ```
 
+## Logging
+
+In order to debug what's happening with the Ads Loading, OpenAds uses [LogLevel](https://github.com/pimterry/loglevel) as it's logging framework.
```

Comment:
We should avoid contractions ('s) in a formal text:

"In order to debug what is happening with Ads Loading, OpenAds uses [LogLevel](https://github.com/pimterry/loglevel) as its logging framework."

Reply:
:+1: